contracts/Vault.sol

```diff
@@ -106,7 +106,7 @@ contract Vault is IVault, Auth {
      **/
     function spawn(address asset, address user, uint _oracleType) external override hasVaultAccess notLiquidating(asset, user) {
         oracleType[asset][user] = _oracleType;
-        delete liquidationBlock[asset][user];
+        delete liquidationTs[asset][user];
```

Comment:
The line is excess, isn't it?

Reply:
yes, since we have modifier `notLiquidating` here